functions-and-scope.js

```diff
@@ -71,7 +127,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 9
+console.log("------------Opdracht 3a-----------------");
+
+let max = 0;
```

Comment:
Wanneer je met een for-loop gaat bepalen wat de hoogste (of laagste) waarde is, dan is het beter om hier de eerste waarde uit de array te pakken
```suggestion
let max = grades[0];
```

Met rapportcijfers maakt het niet zo veel uit, maar stel je voor dat je deze functie had aangeroepen met de array `[-1, -3, -8, -4]`. Dan had de uitkomst van deze functie `0` geweest terwijl die waarde niet eens in de array voorkomt.